modules/ROOT/pages/analytics-event-api.adoc

```diff
@@ -4,54 +4,60 @@ include::_attributes.adoc[]
 
 == Limitations
 
-* Event API is designed to create ad hoc reports, and it is not intended as a data export tool. If you need to export data, keep in mind the following recommendations:
+The Event API is designed to create ad-hoc reports, and is not intended as a data export tool. Before you export data, review the following recommendations:
+
 ** Do not allow calls to overlap time intervals.
-** Retrieve all the data that you need in a single request. Do not repeat the same request for every API that you have.
-** Do not call Event API more than once per minute.
-** For organizations whose API usage exceed 20 000 calls per 10 minutes, consider a different way for data export. API gateway can be configured to send usage events directly to monitoring systems.
-* Event API is not meant for data synchronization. +
-If you are interested in data sync for events with an external system, refer to xref:analytics-event-forward.adoc[Analytics Event Forwarding] for more information.
+** Retrieve all the data that you need in a single request. 
+** Do not repeat the same request for every API that you have.
+** Do not call the Analytics Event API more than once per minute.
+** For organizations whose API usage exceeds 20,000 calls per 10 minutes, consider a different way for data export. 
++
+You can configure API gateway to send usage events directly to monitoring systems.
+* Do not use the Analytics Event API for data synchronization. +
+If you need to synchronize data for events with an external system, see xref:analytics-event-forward.adoc[analytics event forwarding] for more information.
 
 == Creating a Report
 
-The following procedure describes how to create a report to find the API events that, for example, resulted in a policy violation. Suppose you want to find any events that were rejected because of policy violations, such as rate limiting or IP blacklisting. In step 5, you select Violated Policy Name in Fields to filter the data in this manner.
+The following procedure describes how to create a report to find the API events. For example, you might want to find events that were rejected because of policy violations, such as rate limiting or IP blacklisting. 
 
-. In API Manager > Analytics > Manage Reports, click New to create a new report.
-. In Create Report, select a data source, range, and format.
-. In Fields, select Violated Policy Name.
+To create a report:
+
+. In *API Manager > Analytics > Manage Reports*, click *New* to create a new report.
+. In *Create Report*, select a data source, range, and format.
+. In *Fields*, select *Violated Policy Name*.
 +
 A URL appears that you can use to see the report after you save it. The date that data becomes
 available for the data source you specified appears.
 +
 . Save the report.
 +
-When the report becomes available, go to the URL to view it.
+When the report is generated, go to the URL to view it.
```

Comment:
After the user saves the report, is this the point at which it can take up to 10 minutes for results?
How will the user know "when the report is generated"?

Reply:
Done! Thanks!